tests/python/test_transformer_engine.py

```diff
@@ -2,16 +2,23 @@
 # All rights reserved.
 # SPDX-License-Identifier: BSD-3-Clause
 
-# This test is yet to be added to CI. For now, run it manually with `mpirun -n
-# 2 pytest transformer_engine_test.py`.
-
 import os
+import pytest
 import torch
 import torch.distributed as dist
+
 import transformer_engine.pytorch as te
 
+import multidevice
+
+
+@pytest.fixture
```

Comment:
naive question for my own education, is it required to have each test file define fixture inside?

Asking since we are constructing the same fixture in `test_multidevice.py`, I'm wondering if this should be moved inside `multidevice.py` instead?